### ISSUE TITLE
Embodiment dictates learnability in neural controllers. (arXiv:1910.07487v2 [cs.LG] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#6672]
Embodiment dictates learnability in neural controllers. (arXiv:1910.07487v2 [cs.LG] UPDATED)<br>
https://ift.tt/3urQ1Xn<br>
<p>Catastrophic forgetting continues to severely restrict the learnability of controllers suitable for multiple task environments. Efforts to combat catastrophic forgetting reported in the literature to date have focused on how control systems can be updated more rapidly, hastening their adjustment from good initial settings to new environments, or more circumspectly, suppressing their ability to overfit to any one environment. When using robots, the environment includes the robot's own body, its shape and material properties, and how its actuators and sensors are distributed along its mechanical structure. Here we demonstrate for the first time how one such design decision (sensor placement) can alter the landscape of the loss function itself, either expanding or shrinking the weight manifolds containing suitable controllers for each individual task, thus increasing or decreasing their probability of overlap across tasks, and thus reducing or inducing the potential for catastrophic forgetting.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
